.semaphore/semaphore.yml

```diff
@@ -46,4 +46,4 @@ blocks:
             - bundle exec rubocop
       env_vars:
         - name: RUBY_VERSION
-          value: 2.6.5
+          value: 3.0.4
```

Comment:
I figured the semaphore builds should be using the same ruby version as described in .ruby-version